src/components/UnifiedSidenav/UnifiedSidenav.js

```diff
@@ -0,0 +1,217 @@
+  // these are the tab items that we dont need to show in the second pane but need to go through recursively
+  if (isTab) {
+    return (
+      <>
+        {items?.map((tocItem) => (
+          <UnifiedTocNavItem {...tocItem} level={level} slug={slug} />
+        ))}
+      </>
+    );
+  }
+
+  // groups are for adding a static header, these can also be collapsible
+  if (group) {
+    return (
+      <SideNavGroup header={label} collapsible={collapsible} className={cx(sideNavGroupTOCStyling({ level }))}>
+        {items?.map((tocItem) => (
+          <UnifiedTocNavItem {...tocItem} level={level} slug={slug} />
+        ))}
+      </SideNavGroup>
+    );
+  }
+
+  // collapsible is for items that have nested links
+  if (collapsible) {
+    return (
+      <CollapsibleNavItem
+        items={items}
+        label={label}
+        url={url}
+        level={level}
+        slug={slug}
+        className={cx(sideNavItemTOCStyling({ level }))}
+      />
+    );
+  }
+
+  return (
+    <SideNavItem
+      active={isSelectedTab(url, slug)}
+      aria-label={label}
+      as={Link}
+      to={url}
+      className={cx(sideNavItemTOCStyling({ level }))}
+    >
+      {label}
+    </SideNavItem>
+  );
+}
+
+function StaticNavItem({ label, url, glyph, slug, level = 1 }) {
+  return (
+    <SideNavItem
+      active={isSelectedTab(url, slug)}
+      glyph={<Icon glyph={glyph} />}
+      aria-label={label}
+      as={Link}
+      to={url}
+      className={cx(sideNavItemTOCStyling({ level }))}
+    >
+      {label}
+    </SideNavItem>
+  );
+}
+
+// This checks what sidenav should load based on the active Tab
+const isActiveTocNode = (currentUrl, slug, children) => {
+  if (currentUrl === undefined) return false;
+  if (isCurrentPage(currentUrl, slug)) return true;
+  if (children) {
+    return children.reduce((a, b) => a || isActiveTocNode(currentUrl, b.url, b.items), false);
+  }
+  return false;
+};
+
+export function UnifiedSidenav({ slug }) {
+  const unifiedTocTree = useUnifiedToc();
+  const staticToc = unifiedTocTree.filter((item) => item?.isTab);
+
+  const [activeTabUrl, setActiveTabUrl] = useState(() => {
+    const activeToc = staticToc.find((staticToc) => {
+      return isActiveTocNode(slug, staticToc.url, staticToc.items);
+    });
+    return activeToc?.url;
+  });
+
+  useEffect(() => {
+    setActiveTabUrl(() => {
+      const activeToc = staticToc.find((staticToc) => {
+        return isActiveTocNode(slug, staticToc.url, staticToc.items);
+      });
+      return activeToc?.url;
+    });
+  }, [slug, staticToc]);
+
+  // Hide the Sidenav with css while keeping state as open/not collapsed.
+  // This prevents LG's SideNav component from being seen in its collapsed state on mobile
+  return (
+    <>
+      <SideNav widthOverride={400} className={cx(sideNavStyle)} aria-label="Bianca's Side navigation">
+        <div className={cx(leftPane)}>
+          {staticToc.map((navItems) => {
```

Comment:
[minor nit] argument navItems sounds like a list
```suggestion
          {staticTocItems.map((staticTocItem) => {
```

or replace `staticTocItems` with `staticTocs`

Reply:
fixed!